react-backend/client/src/shared/components/layout/base.js

```diff
@@ -13,4 +13,5 @@ export const FlexContainerCol = styled(FlexContainer)`
 export const FlexContainerRow = styled(FlexContainer) `
   flex-direction: row;
   flex-wrap: wrap;
+  ${props => (props.flex ? css`flex: ${props.flex};` : '')}
```

Comment:
What does this do?

Reply:
It sets the `flex` property only if the `flex` prop is set